sbml/RECON1/reactions.yaml

```diff
@@ -10018,7 +10018,7 @@
   equation: atp[n] + ump[n] <=> adp[n] + udp[n]
 - id: UNK2
   name: UNK2
-  equation: 2kmb[c] + gln__L[c] + h[c] => glu__L[c] + met__L[c]
+  equation: M_2kmb[c] + M_gln__L[c] + (2) M_h_c[c] => M_glu__L[c] + M_met__L[c]
```

Comment:
@ssuchacz I think the `M_` prefix, and on `M_h_c` the `_c` suffix, should not be added. These prefixes/suffixes are no longer part of the model so your change should not reintroduce them. The change here should just be to add the additional `h[c]`.